.github/workflows/tests.yml

```diff
@@ -3,30 +3,42 @@ name: Build tests
 on: [push, pull_request]
 
 env:
+  # Minimum userspace version to build refpolicy.
   SELINUX_USERSPACE_VERSION: checkpolicy-3.1
 
 jobs:
   lint:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     # This version should be the minimum required to run the fc checker
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: 3.7
 
     - name: Install dependencies
       run: |
-        sudo apt-get update -qq
+        sudo apt-get update -q
+        sudo apt-get install -qy autoconf-archive bison flex libconfuse-dev uthash-dev
 
-        # Install SELint from Debian testing
-        wget -O - https://ftp-master.debian.org/keys/archive-key-10.asc 2>/dev/null | sudo apt-key add -
-        sudo add-apt-repository 'deb http://deb.debian.org/debian/ testing main' -y
-        sudo apt-get install -qqy selint
-        selint -V
+    - name: Checkout SELint
+      uses: actions/checkout@v3
+      with:
+        repository: SELinuxProject/selint
+        # support exclusions in interface arguments
+        ref: 'v1.3.0' # "Parse interface taking list of exemptions"
```

Comment:
The comment referred to the old ref, and no longer matches the tag.  Given that the tag is somewhat more descriptive than a ref, the comment can probably be dropped.